js/client-teambuilder.js

```diff
@@ -1699,19 +1699,19 @@
 
 			if (template.speciesid === 'meowstic') {
 				smogdexid = 'meowstic-m';
-			} else if (smogdexid === 'rotom' || smogdexid === 'deoxys' || smogdexid === 'kyurem' || smogdexid === 'giratina' || smogdexid === 'shaymin' || smogdexid === 'tornadus' || smogdexid === 'thundurus' || smogdexid === 'landorus' || smogdexid === 'pumpkaboo' || smogdexid === 'gourgeist' || smogdexid === 'arceus' || smogdexid === 'meowstic' || smogdexid === 'hoopa') {
+			} else if (smogdexid === 'rotom' || smogdexid === 'deoxys' || smogdexid === 'kyurem' || smogdexid === 'giratina' || smogdexid === 'shaymin' || smogdexid === 'tornadus' || smogdexid === 'thundurus' || smogdexid === 'landorus' || smogdexid === 'pumpkaboo' || smogdexid === 'gourgeist' || smogdexid === 'arceus' || smogdexid === 'meowstic' || smogdexid === 'hoopa' || smogdexid === 'lycanroc' || smogdexid === 'diglett' || smogdexid === 'dugtrio' || smogdexid === 'exeggutor' || smogdexid === 'geodude' || smogdexid === 'graveler' || smogdexid === 'golem' || smogdexid === 'grimer' || smogdexid === 'muk' || smogdexid === 'marowak' || smogdexid === 'meowth' || smogdexid === 'persian' || smogdexid === 'vulpix' || smogdexid === 'ninetales' || smogdexid === 'sandshrew' || smogdexid === 'sandslash' || smogdexid === 'raichu' || smogdexid === 'rattata' || smogdexid === 'raticate' || smogdexid === 'oricorio' || smogdexid === 'silvally' || smogdexid === 'zygarde') {
```

Comment:
May be easier to detect the forme name rather than listing all the Alola Pokémon.